src/attributes.jl

```diff
@@ -183,6 +183,13 @@ end
 
 ## Instance attributes
 
+"""
+    ListOfInstanceAttributesSet()
+
+A `Vector{AbstractInstanceAttribute}` of all instance attributes that were set to the instance and are not instance-specific.
```

Comment:
What does `instance-specific` mean?

Reply:
`solver-specific` only makes sense for solver instance so I used `instance-specific` which should make sense for any `AbstractInstance`

Reply:
What is the definition of instance-specific? Anything that is not defined directly in MOI?

Reply:
If so, this definition does not work. An instance may have an attribute that's essential to the definition of the problem that's not defined in MOI. Imagine an MOI extension for multiobjective problems.